BrowserKit/Sources/SummarizeKit/SummarizeController.swift

```diff
@@ -6,14 +6,38 @@ import Foundation
 import Common
 import UIKit
 import ComponentLibrary
+import MarkdownKit
 
-// swiftlint:disable line_length
+// swiftlint:disable all
```

Comment:
@FilippoZazzeroni this is just for the dummy text really. The linter was complaining about vertical spaces and whatnot and these are needed in markdown syntax. `dummyText` will be removed soon.